core/unit_test/CMakeLists.txt

```diff
@@ -545,7 +569,14 @@ if(Kokkos_ENABLE_CUDA)
       cuda/TestCuda_TeamScratchStreams.cpp
       ${Cuda_SOURCES3}
       cuda/TestCuda_Spaces.cpp
-  )
+    )
+
+    KOKKOS_ADD_EXECUTABLE_AND_TEST(
+    UnitTest_Cuda4
```

Comment:
Any good reason to introduce a new executable for that test?

Reply:
Originated from the time of allocating large chunks of memory. Will add it to another exec

Reply:
done in [10c9dc](https://github.com/kokkos/kokkos/pull/5289/commits/10c9dcf1fad358cf53df4e5fc3e09d1d72b7ac7f)